src/index.mjs

```diff
@@ -173,14 +201,19 @@ async function getTargetLanguages() {
 }
 
 // The type assertion below asserts that the API will return a single result, not an array of results
-async function translateText ({ text, targetLangCode, formality }) {
+async function translateText ({ text, targetLangCode, formality, glossaryId }) {
   // Validate languages before translation
   targetLanguages.validate(targetLangCode);
 
   try {
-    const result = await deeplClient.translateText(text, null, targetLangCode, { formality });
+    const options = { formality };
```

Comment:
I guess you could also do something like
```
const options = { 
  formality,
  ...(glossaryId && { glossary: glossaryId })
};
```

but your version is very clear. I just wish there was something like the `?` operator here ;)